### PR TITLE
added fonts and fixed trip list and footer style

### PR DESCRIPTION
**Pull request by marobo**

### CODE REVIEW

File: map/templates/map/mapview.html [hunk 1]
```diff
@@ -35,42 +35,51 @@
...
+                {% for via in viazen %}
+                    {% if via.creator %}
+                    <div class="each-upload">
+                        <p><img src="{% static 'img/mario.jpeg' %}" class="rounded-circle" alt="Cinque Terre" width="40" height="40">
+                            <strong class="person-name">{{ via.creator }}</strong>
+																						  </p>
+                        <strong>{{ via.title }}</strong>
+                        <div class="gallery">
+                            <figure class="gallery__item gallery__item--3">
+                                <img src='{{ via.image_trip.url }}' class="gallery__img" width="300" height="240" alt="Image 3">
```
Comment (eraulo) [inline map/templates/map/mapview.html:65]: @marobo Can you change the width="300" height="240" to width="293" height="293"?

File: map/templates/map/mapview.html [hunk 2]
```diff
@@ -35,42 +35,51 @@
...
+            <div class="col-md-2 add-journey">
                 <a href={% url 'hatama_viajen' %} class="btn btn-primary add-journey">Add my journey</a>
             </div>
         </div>
         <div class="row">
             <div class="col-md-2 list-users">
-                {%for user in users %}
-                    <span class="person-name">{{ user.first_name }}</span></p>
-                {% endfor %}
+            {% for user in users %}
```
Comment (eraulo) [inline map/templates/map/mapview.html:51]:
```suggestion
            {% for user in users %}
             {% if use.first_name %}
               {{ user.first_name }}
              {% else %}
              {{ user.username }}
             {% endif %}
             {% endfor %} 
             
```

File: map/static/css/timor.css [hunk 3]
```diff
@@ -1,7 +1,18 @@
+@font-face {
+    font-family: 'Montserrat';
+    src: url(../fonts/Montserrat-Regular.woff2);
+}
+
+@font-face {
```
Comment (eraulo) [inline map/static/css/timor.css:6]: Why is double @font-face?

File: map/static/css/timor.css [hunk 4]
```diff
@@ -1,7 +1,18 @@
...
+    font-family: 'Comfortaa';
+    src: url(../fonts/Comfortaa_Thin.woff2);
+}
+
 body {
     margin: 0px;
     color: #000000c7;
     font-family: 'Montserrat';
+    background-color: #f0f2f5;
```
Comment (eraulo) [inline map/static/css/timor.css:15]:
```suggestion
    background-color: #f0f2f5 to `#fafafa`
```

File: map/templates/map/mapview.html [hunk 5]
```diff
@@ -35,42 +35,51 @@
...
+                            <strong class="person-name">{{ via.creator }}</strong>
+																						  </p>
+                        <strong>{{ via.title }}</strong>
+                        <div class="gallery">
+                            <figure class="gallery__item gallery__item--3">
+                                <img src='{{ via.image_trip.url }}' class="gallery__img" width="300" height="240" alt="Image 3">
+                            </figure>
+                        </div>
+                        <p>{{ via.description }}</p>
+                        <p class="text-muted h6">Durations | April 5, 2020, noon - April 7, 2020 noon</p>
```
Comment (eraulo) [inline map/templates/map/mapview.html:69]:
```suggestion
                        <p class="text-muted h6"> April 5, 2020, noon - April 7, 2020 noon</p>
```

File: map/templates/map/mapview.html (hunk 1 above)
Comment (marobo) [reply map/templates/map/mapview.html:65]: bele...

File: map/templates/map/mapview.html (hunk 2 above)
Comment (marobo) [reply map/templates/map/mapview.html:51]: Tamba saida mak ita tenki halakon lina nee?
 `<span class="person-name">{{ user.first_name }}</span></p>`
No hau koko run ita nia sujestaun maibe hau la hetan first name maske hau registu kompletu ho first name.

File: map/static/css/timor.css (hunk 3 above)
Comment (marobo) [reply map/static/css/timor.css:6]: hau uza font-family rua, `Montserrat` ho `Comfortaa`, 
Montserrat uza ba default ka iha body no Comfortaa ita uza iha titulu sira!
Comfortaa nia exemplu mak iha screenshot nee 
image

File: map/static/css/timor.css (hunk 4 above)
Comment (marobo) [reply map/static/css/timor.css:15]: why?

File: map/static/css/timor.css (hunk 3 above)
Comment (eraulo) [reply map/static/css/timor.css:6]: Ok diak

File: map/static/css/timor.css (hunk 4 above)
Comment (eraulo) [reply map/static/css/timor.css:15]: Because this is color isn't good to see.

Comment (marobo) [reply map/static/css/timor.css:15]: background-color nee hanesan fali ho header nian,,
se ita halo suggestion hanesan neba bainhira ita klik commit changes nee sei replace ho nee:  "to `#fafafa`" ,

File: map/templates/map/mapview.html [hunk 6]
```diff
@@ -35,42 +35,75 @@
...
         </div>
         <div class="row">
             <div class="col-md-2 list-users">
-                {%for user in users %}
+                {% for user in users %}
+                    {% if user.first_name %}
                     <span class="person-name">{{ user.first_name }}</span></p>
+                    {% else %}
+                    <span class="person-name">{{ user.username }}</span></p>
+                    {% endif %}
```
Comment (PeteCoward) [inline map/templates/map/mapview.html:56]:
```suggestion
                    <span class="person-name">{{ user.first_name|default:user.username }}</span>
```

File: map/templates/map/mapview.html [hunk 7]
```diff
@@ -35,42 +35,75 @@
...
+                                {% endif %}
+                            {% endfor %}
+                            <figure class="gallery__item gallery__item--3">
+                                <a href="{% url "photo_viazen" via.id %}">
+                                    <img src="{% static 'img/photo_add.png' %}" class="gallery__img icon_upload" align="center" width="200px" height="200px" alt="Image icon">
+                                </a>
+                            </figure>
+                        </div>
+                        <p>{{ via.description }}</p>
+                        <p class="text-muted h6">April 5, 2020, noon - April 7, 2020 noon</p>
```
Comment (PeteCoward) [inline map/templates/map/mapview.html:93]:
```suggestion
```

**Pull request merged**